### DISCUSSION TITLE
Actions Monorepo Support

### DISCUSSION THREAD

**Discussion by rawkode** [community/community#9537] [Actions]
I’d prefer for each project within my monorepo to have its own `.github` directory and workflows, which will be executed based on the paths modified within the commit. 

While we can do this manually with lots of workflow files at the top level and using `on:path`, it’s rather cumbersome and messy 

i believe this would be a great quality of life improvement for us monorepo users

**Comment by ethomson** [comment]
Thanks @rawkode for the feedback.  I agree that there's some improvements that we could make for monorepos, I'll put this on our backlog.

**Comment by zoro16** [comment]
Is there any update on this feature request ? @ethomson
